### ISSUE TITLE
RDS v3 instance does not pick up Region from Provider

### ISSUE THREAD

**Issue by netclectic** [opentelekomcloud/terraform-provider-opentelekomcloud#1951]
When creating a RDS instance with the `opentelekomcloud_rds_instance_v3` resource, the `region` parameter is not being picked up from the value configured in the provider. 

# Terraform provider version
- Terraform v1.3.0
- windows_amd64
- provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.31.4

# Affected Resource(s)
- opentelekomcloud_rds_instance_v3

# Terraform Configuration Files
- https://gist.github.com/netclectic/0c9124c7652fa18c50706ce2c779c350

# Debug Output/Panic Output
- https://gist.github.com/netclectic/6226cb441088d74e58a86ad0e604221a

# Steps to Reproduce
1. `terraform apply`

# Expected Behavior
RDS instance was created

# Actual Behavior
Exception creating RDS: `error creating new RDSv3 instance: Missing input for argument [Region]`

# Important Factoids
Actually trying to do this through CDKTF, but the result is the same when executing directly with Terraform 

# References
The [config source](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/e2ac6c29d75d39beabead13c8f7072029334ddbe/opentelekomcloud/common/cfg/config.go#L1002) suggests the region should be picked up from the provider:
```go
// GetRegion returns the region that was specified in the resource. If a
// region was not set, the provider-level region is checked. The provider-level
// region can either be set by the region argument or by OS_REGION_NAME.
```

- there is no specific `region` property available for the `opentelekomcloud_rds_instance_v3` resource
- the region is specified for the provider, but is not picked up when creating the RDS instance
- if I specify the `OS_REGION_NAME` environment variable, then this is picked up and the RDS instance is created as expected

**Comment by artem-lifshits** [comment]
Hello @netclectic I'm trying to reproduce the issue but no luck so far.
Would you be so kind to provide your Terraform code so I could check directly in TF?
Thanks.

**Comment by netclectic** [comment]
I made a test repo - https://github.com/netclectic/TerraformOTCRDSv3Test

Obviously, you'll need to replace various entries in variables.tf

**Comment by artem-lifshits** [comment]
Hello @netclectic the issue should be fixed in `v1.31.6`, could you please check?
Thank you.

**Comment by netclectic** [comment]
Yep, working now.

Thanks!
